scripts/update_birdnet.sh

```diff
@@ -15,7 +15,7 @@ while getopts ":r:b:" o; do
       remote=${OPTARG}
 
       # Ensure that git remote is configured
-      git remote show $remote 2&> /dev/null || {
+      git remote show $remote 1&> /dev/null || {
```

Comment:
I see now -- no `test` is actually envoked, which is why you have to retain the stderr for the `||` logic to work.

If you put it in a test I think you can dispense with all output
```
if git remote show $remote &>/dev/null;then
  remote=${OPTARG}
else
  echo "Error: remote '$remote' not found. Add the upstream remote to your repository and try again."
  exit 1
fi
```